server/graphql/v2/mutation/AccountMutations.ts

```diff
@@ -455,6 +528,16 @@ const accountMutations = {
         );
       }
 
+      models.Activity.create({
+        type: activities.COLLECTIVE_DELETED,
+        UserId: req.remoteUser.id,
+        UserTokenId: req.userToken?.id,
+        CollectiveId: account.id,
+        FromCollectiveId: account.id,
+        HostCollectiveId: account.approvedAt ? account.HostCollectiveId : null,
+        data: { name: account.name, slug: account.slug },
+      });
+
```

Comment:
Can we do that after `deleteCollective`? `deleteCollective` can fail

Reply:
Done. 👍🏽